### PR TITLE
update true==1 for cppbuild.sh

### PR DESCRIPTION
**Pull request by jbkyang-nvi**
change logic for true/false in cppbuild

### CODE REVIEW

File: tritonserver/cppbuild.sh [hunk 1]
```diff
@@ -7,13 +7,13 @@ if [[ -z "$PLATFORM" ]]; then
...
-INCLUDE_DEVELOPER_TOOLS_SERVER=${INCLUDE_DEVELOPER_TOOLS_SERVER:=0}
+INCLUDE_DEVELOPER_TOOLS_SERVER=${INCLUDE_DEVELOPER_TOOLS_SERVER:=1}
 
-if [[ ! -f "/opt/tritonserver/include/triton/developer_tools/generic_server_wrapper.h" ]] && [[ ! -f "/opt/tritonserver/lib/libtritondevelopertoolsserver.so" ]] && [[ ${INCLUDE_DEVELOPER_TOOLS_SERVER} -eq 0 ]]; then
+if [[ ! -f "/opt/tritonserver/include/triton/developer_tools/generic_server_wrapper.h" ]] && [[ ! -f "/opt/tritonserver/lib/libtritondevelopertoolsserver.so" ]] && [[ ${INCLUDE_DEVELOPER_TOOLS_SERVER} -eq 1 ]]; then
     TOOLS_BRANCH=${TOOLS_BRANCH:="https://github.com/triton-inference-server/developer_tools.git"}
     TOOLS_BRANCH_TAG=${TOOLS_BRANCH_TAG:="main"}
     TRITON_CORE_REPO=${TRITON_CORE_REPO:="https://github.com/triton-inference-server/core.git"}
-    TRITON_CORE_REPO_TAG=${TRITON_CORE_REPO_TAG="r23.05"}
+    TRITON_CORE_REPO_TAG=${TRITON_CORE_REPO_TAG="main"}
```
Comment (saudet) [inline tritonserver/cppbuild.sh:16]: Let's keep a version that matches with the pom.xml file here.

Comment (saudet) [reply tritonserver/cppbuild.sh]: We can probably update everything to 2.38.0/23.09 and the presets should still work. Could you give it a try?

**Comment by jbkyang-nvi** [comment]
@saudet Did the changes. It seems like its failing on finding cmake. It doesn't seem like my changes will effect it. Am I missing something?

**Comment by saudet** [comment]
> @saudet Did the changes. It seems like its failing on finding cmake. It doesn't seem like my changes will effect it. Am I missing something?

It's fixed now, sorry about that.

**Review by saudet — APPROVED** [review]

**Comment by jbkyang-nvi** [comment]
Thanks Samuel! Look forward to seeing it merged 🙏

**Pull request merged**